src/main/kotlin/com/github/minecraftschurlimods/helperplugin/HelperExtension.kt

```diff
@@ -171,21 +169,10 @@ open class HelperExtension @Inject constructor(private val project: Project) {
         LIBRARY("GAMELIBRARY")
     }
 
-    private val neoforgeDependency = neoVersion.map { project.dependencyFactory.create("net.neoforged", "neoforge", it) }
     private val testframeworkDependency = neoVersion.map { project.dependencyFactory.create("net.neoforged", "testframework", it) }
 
-    fun neoforge() = neoforgeDependency
     fun testframework() = testframeworkDependency
 
-    fun withJarJar() {
-        project.tasks.named<Jar>("jar") {
-            archiveClassifier.set("slim")
```

Comment:
Note: I removed this because MDG will add the jarjar stuff to the normal jar directly. It is possible to create a slim jar manually but it shouldn't be necessary.